setup-jenkins-on-cent-os-7.sh

```diff
@@ -11,9 +11,9 @@
 # INSTRUCTIONS FOR USE:
 # 1. Copy this shell script to your home directory or the /tmp directory.
 # 2. Make it executable with the following command: 
-#      chmod a+x setup-jenkins-on-cent-os-7.sh
+  #   chmod a+x setup-jenkins-on-cent-os-7.sh
```

Comment:
Hi! I can't tell what you want to change other than a slight bit of white space. If it's for readability purposes, I don't see it adding much value in that way.